CHANGELOG.md

```diff
@@ -28,6 +27,21 @@
 
 ## 1.9.1-RELEASE (2021-11-09)
 
```

Comment:
This doesn't look right.. there should be a new entry for AFK feature but also there shouldn't be any changes compared to develop but here I see 1.9.1 - 1.9.3 changes